snuba/web/rpc/common/eap_transform.py

```diff
@@ -0,0 +1,65 @@
+from typing import Sequence
+
+from snuba.query import Column, Expression, Query, SelectedExpression
+from snuba.query.expressions import FunctionCall
+from snuba.query.processors.logical.hash_bucket_functions import (
+    HashBucketFunctionTransformer,
+)
+from snuba.query.query_settings import HTTPQuerySettings
+
+HASH_BUCKET_COLUMNS = ["attr_str", "attr_num"]
+
+functions_transformer = HashBucketFunctionTransformer(HASH_BUCKET_COLUMNS)
+
+
+def replace_timestamp(exp: Expression) -> Expression:
+    if not isinstance(exp, Column):
+        return exp
+
+    if exp.column_name == "timestamp":
+        return Column(
+            table_name=exp.table_name, column_name="_sort_timestamp", alias=exp.alias
+        )
+
+    return exp
+
+
+def replace_selected_column(
```

Comment:
@volokluev is there a way of doing this more nicely?

Reply:
what is this for?